src/test/java/com/redhat/cloud/notifications/events/LifecycleITest.java

```diff
@@ -97,24 +107,25 @@ void setup() {
     @Test
     void t00_setupBundle() {
         Bundle bundle = new Bundle(BUNDLE_NAME, "A bundle");
-        theBundle =
-            given()
+        Response response = given()
                 .body(bundle)
                 .contentType(ContentType.JSON)
                 .basePath("/")
-            .when()
-                    .post("/internal/bundles")
-            .then()
+                .when()
```

Comment:
I know IDEs don't really like the mixed indent. I like it when `given()`, `when()`, ` then()` are less indented as this sort of shows the control structure as with normal if/then/else blocks

Reply:
Unfortunately I didn't change that one on purpose, that's IntelliJ's automatic formatting.

Reply:
I know - this is why I say IDEs don't like that